test/studies/shootout/meteor/kbrady/meteor-parallel.chpl

```diff
@@ -344,11 +348,11 @@ module meteor {
     for row1 in 0..31 {
       for row2 in 0..31 {
         for row3 in 0..31 {
-          var idx = row1+(row2*32)+(row3*1024);
+          const idx = row1+(row2*32)+(row3*1024);
```

Comment:
Not sure how I missed this, but I'd be tempted to add spaces between the "+" and each operand.